templates/postgresql_database.yml.erb

```diff
@@ -4,9 +4,19 @@ development: &default
   encoding: utf8
   host: localhost
   min_messages: warning
-  pool: 2
+  pool: <%%= ENV.fetch("DB_POOL", 2) %>
+  reaping_frequency: <%%= ENV.fetch("DB_REAPING_FREQUENCY", 10) %>
   timeout: 5000
 
 test:
   <<: *default
   database: <%= app_name %>_test
+
+production: &deploy
+  encoding: utf8
+  min_messages: warning
+  pool: <%%= [ENV.fetch("MAX_THREADS", 2), ENV.fetch("DB_POOL", 2)].max %>
```

Comment:
Curious why we'd default to `MAX_THREADS` or `2` for the database pool, while defaulting each Puma worker to `MAX_THREADS` or `5`?